src/js/PhotoSphereViewer.core.js

```diff
@@ -74,21 +74,32 @@ PhotoSphereViewer.prototype._loadXMP = function() {
     throw new PSVError('Cannot load image');
   };
 
-  xhr.open('GET', this.config.panorama, true);
-  xhr.send(null);
-
+  if (true === this.prop.cacheTextures && 'undefined' != typeof this.prop.loadedXMP[this.config.panorama]) {
```

Comment:
didn't you mixed "prop" and "config" ? seems like this is a user parameter => "config"